app/models/user.rb

```diff
@@ -1,3 +1,6 @@
+require 'net/ldap'
+require 'fakeldap'
```

Comment:
why does the user model need fakeldap?

Reply:
@mcritchlow - The user model doesn't need fakeldap.  I added it in while testing and forgot to remove it.  I'll remove it and re-commit.